Tests/Helpers/FakeIlpPaymentNetworkClient+Test.swift

```diff
@@ -1,8 +1,34 @@
+import SwiftGRPC
 import XpringKit
 
 extension FakeIlpPaymentNetworkClient {
     /// A network client that always succeeds
-    static let successfulFakeNetworkClient = FakeIlpPaymentNetworkClient(
-        sendPaymentResult: .success(.testSendPaymentResponse)
-    )
+    static let successfulFakeNetworkClient = FakeIlpPaymentNetworkClient(sendPaymentResult: .success(.testSendPaymentResponse))
+
+    /// Creates a FakeIlpPaymentNetworkClient which will always throw the specified Error
+    ///
+    /// - Parameters:
+    ///     - errorResponse: An Error that will be thrown by the network client
+    public static func withErrorResponse(_ errorResponse: Error) -> FakeIlpPaymentNetworkClient {
```

Comment:
optional nit: I think it's slightly more idiomatic to call this:
```
with(errorResponse: Error) 
```

But I can't explicitly find anything in the Swift naming guide on this:
https://swift.org/documentation/api-design-guidelines/

But I can't